templates/repo/settings/options.tmpl

```diff
@@ -16,9 +16,9 @@
 					<label for="repo_name">{{.locale.Tr "repo.repo_name"}}</label>
 					<input id="repo_name" name="repo_name" value="{{.Repository.Name}}" data-repo-name="{{.Repository.Name}}" autofocus required>
 				</div>
-				<div class="inline field">
+				<div class="inline field" {{if not (eq .Repository.LFSSize 0)}}title="git: {{FileSize .Repository.Size}}, lfs: {{FileSize .Repository.LFSSize}}"{{end}}>
```

Comment:
```suggestion
				<div class="inline field{{if not (eq .Repository.LFSSize 0)}} tooltip{{end}}"{{if not (eq .Repository.LFSSize 0)}} data-content="git: {{FileSize .Repository.Size}}, lfs: {{FileSize .Repository.LFSSize}}"{{end}}>
```

use tippy tooltip instead of title.

Reply:
that's okay, thanks
![image](https://user-images.githubusercontent.com/25342410/219011704-d5c038cc-740a-4f0b-9f31-9d493dfcc4b5.png)